onevizion/integration/log.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import json
+from onevizion.util import *
+from onevizion.curl import curl
+from onevizion.integration.loglevel import 
```

Comment:
Looks like a typo 
I think it should be ```from onevizion.integration.loglevel import LogLevel```

Reply:
Yes. Done